Directory.Build.targets

```diff
@@ -0,0 +1,39 @@
+<!--
+  Everything in this file should have conditions that check $(RepoRoot). Similarly, because targets here could
+  unconditionally override dotnet/aspnetcore targets, put them in a conditionally-imported .targets file instead.
+-->
+<Project>
+  <!--
+    When outside dotnet/aspnetcore, intentionally break the inheritance chain.
+    Isolate this repo from irrelevant settings in parent folders.
+  -->
+  <Import Project="$(RepoRoot)src/ProjectTemplates/Directory.Build.targets" Condition=" '$(RepoRoot)' != '' " />
+
+  <!-- Copied from https://github.com/dotnet/aspnetcore/blob/0c5456afe95096db6ac5400cecd3440299b42714/src/ProjectTemplates/Directory.Build.targets. -->
+  <PropertyGroup Condition=" '$(RepoRoot)' == '' ">
+    <PackageVersion Condition=" '$(PackageVersion)' == '' ">$(Version)</PackageVersion>
```

Comment:
NuGet defaults to producing a package named Microsoft.DotNet.Web.Spa.ProjectTemplates.6.0.1.0.0.nupkg. Could hard-code `$(Version)` and `$(PackageVersion)` to be `0.1.0` or `0.1.0-dev` if anyone prefers.